packages/protocol/contracts/L1/tiers/TaikoA6TierProvider.sol

```diff
@@ -82,8 +87,14 @@ contract TaikoA6TierProvider is ITierProvider {
     }
 
     function getMinTier(uint256 rand) public pure override returns (uint16) {
-        if (rand % 1000 == 0) return LibTiers.TIER_SGX_AND_PSE_ZKEVM;
-        else if (rand % 100 == 0) return LibTiers.TIER_SGX;
+        // If we assume the block time is 3 seconds, and the proof generation
+        // time is ~90 mins
+        // and half of the blocks are unprovable: 90 * 60 / 3 / 2  = 900.
+        // TODO(david): tune this value based on the A6 circuits benchmark.
+        if (rand % 900 == 0) return LibTiers.TIER_SGX_AND_PSE_ZKEVM;
+        else if (rand % 100 == 0) return LibTiers.TIER_SGX; // 1% of the blocks
+            // will be slected to require a SGX proof.
```

Comment:
1% seems low. Why not 100%? For testing purposes?

Reply:
If so, then I think the optimistic tier in this A6 provider can be removed? emmm do we want to do that?

Reply:
Brecht, I think we should not make it 100% to also test the lowest tier. A a prover, we can always submit SGX proofs even if the block requires a lower tier proof.

Reply:
10%?

Reply:
Changed to 10%